packages/odyssey/src/scss/components/_status.scss

```diff
@@ -11,16 +11,17 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+$status-indicator-size: 0.5714285714em;
```

Comment:
Can we move back to using one of our spacing units here?

Reply:
Depends, the only semi workable value I see is `spacing-xs` (6px), but it may be too small. Thoughts @joejezowski-okta? 
![image](https://user-images.githubusercontent.com/62259644/85346097-464ac780-b4a9-11ea-9a0a-081e8cf319ca.png)

Reply:
Hey there…these components weren't updated yet. Sorry about that. I've updated them in Figma and they do use our updated spacing units. See here:

https://www.figma.com/file/6AhmJASP2xMuIQXimwVodo/Odyssey-Refactor?node-id=12%3A143

There is now no space between the label and the actual status. There is a 12px spacing unit between the dot and the status value.